src/SAWScript/Crucible/Common.hs

```diff
@@ -24,7 +24,8 @@ import           Lang.Crucible.Simulator.Profiling
 import           Lang.Crucible.Backend (AbortExecReason(..), ppAbortExecReason, IsSymInterface)
 import           Lang.Crucible.Backend.Online
 import qualified Data.Parameterized.Nonce as Nonce
-import qualified What4.Solver.Yices as Yices
+import qualified What4.Solver.Z3 as Z3
+import qualified What4.Protocol.SMTLib2 as SMT2
```

Comment:
Swapping Z3 for Yices may have some pretty big implications for proofs in-use today. Has any thought been given into parameterizing solver choice here?

Reply:
this is the solver used for path-sat checks, so I agree the change may affect existing proofs. All the existing Amazon and BLST proofs are still passing in CI, so I'm hopeful the impact is limited. Because of internal types, it's not easy to make this code parametric in the solver choice. I will open an issue to track this.

Reply:
https://github.com/GaloisInc/saw-script/issues/1520